kubecost.yaml

```diff
@@ -506,13 +506,15 @@ data:
     upstream api {
         server kubecost-cost-analyzer.kubecost:9001;
     }
-
```

Comment:
Please don't modify `kubecost.yaml` in a feature PR. It has to be updated with specific `helm template` flags and that generally occurs alongside a release of Kubecost.